sql/rename.go

```diff
@@ -203,6 +203,88 @@ func (p *planner) RenameIndex(n *parser.RenameIndex) (planNode, error) {
 
 	idx.Name = newIdxName
 	descKey := MakeDescMetadataKey(tableDesc.GetID())
+	if err := tableDesc.Validate(); err != nil {
```

Comment:
oh, my mistake - i was thinking the addition here was a permission check. sorry!